homeassistant/components/backup/websocket.py

```diff
@@ -306,7 +306,10 @@ async def backup_agents_info(
     connection.send_result(
         msg["id"],
         {
-            "agents": [{"agent_id": agent_id} for agent_id in manager.backup_agents],
+            "agents": [
+                {"agent_id": agent.agent_id, "name": agent.name}
```

Comment:
I'm not sure, if this is exactly what you meant by "Frontend is impacted, from core side we need to send the name" and what needs to be changed on frontend side :grimacing: